flexget/plugins/clients/transmission.py

```diff
@@ -53,7 +54,7 @@ def prepare_config(self, config):
                 logger.error('netrc: {}, file: {}, line: {}', e.msg, e.filename, e.lineno)
         return config
 
-    def create_rpc_client(self, config):
+    def create_rpc_client(self, config) -> transmissionrpc.Client:
```

Comment:
```suggestion
    def create_rpc_client(self, config) -> 'transmissionrpc.Client':
```

This is probably the reason for the test failures.